### DISCUSSION TITLE
Best practices

### DISCUSSION THREAD

**Discussion by typexy** [flyerhq/flutter_chat_ui#175] [Q&A] [answered]
Hi,

I'm asking regarding best practices for this chat app. Let's say I have an app with 200 1:1 chats. Should I create 200x snapshot listener when opening this app to receive update messages? Or what is considered as best practice to receive updates continuously? 

Also, what is considered best practice on when to suspend the realtime listeners?

Furthermore, regarding notifications when app is terminated / in background, I think we have to use e.g. FCM for background notifications about new messages.

**Comment by demchenkoalex** [comment, marked as answer]
Hey, this is not the app, it is just the chat UI. How you handle your backend is totally up to you :)
